tests/fixtures/src/MockNewDiscussionModule.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @author Richard Flynn <richard.flynn@vanillaforums.com>
+ * @copyright 2009-2020 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Fixtures;
+
+/**
+ * A mock new discussion module to test NewDiscussionModule methods.
+ */
+
+class MockNewDiscussionModule {
```

Comment:
In this case you aren't really testing the module. If the module is changed this test won't help.

Maybe you can try `class MockNewDiscussionModule extends NewDiscussionModule` and then override the constructor to not do anything?